crates/revm/src/state_change.rs

```diff
@@ -318,21 +319,22 @@ where
         Err(e) => {
             evm.context.evm.env = previous_env;
             return Err(BlockValidationError::WithdrawalRequestsContractCall {
-                message: e.to_string(),
+                message: format!("execution failed: {e}"),
```

Comment:
`execution failed` should be part of the error

Reply:
not really, we don't have a transaction hash to attach, and it's an irregular state transition, i.e. not a real transaction https://github.com/paradigmxyz/reth/blob/308f979402e638c6ac5494d38230461a2f4fc0d3/crates/interfaces/src/executor.rs#L11-L19